client/landing/stepper/declarative-flow/internals/steps-repository/hundred-year-plan-thank-you/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { useTranslate } from 'i18n-calypso';
+import FormattedHeader from 'calypso/components/formatted-header';
+import HundredYearPlanStepWrapper from '../hundred-year-plan-step-wrapper';
+import type { Step } from '../../types';
+
+import './styles.scss';
+
+const HundredYearPlanThankYou: Step = function HundredYearPlanThankYou( { flow } ) {
+	const translate = useTranslate();
+
+	return (
+		<HundredYearPlanStepWrapper
+			stepContent={
+				<div>
+					<p>TODO: Thank You content</p>
+				</div>
+			}
+			formattedHeader={
+				<FormattedHeader
+					brandFont
+					headerText={ translate( 'TODO: Thank You title' ) }
+					subHeaderText={ translate( 'TODO: Thank You header' ) }
```

Comment:
I don't think we should translate TODO strings. 🤔 
Once we merge this PR, the translation will be triggered.

I think providing the hardcoded strings (without a translate call) is enough until we get the copy with a proper design.

Reply:
Thanks, good call! I've removed the translated strings on all 3 steps.